sway/input/cursor.c

```diff
@@ -69,6 +69,20 @@ struct sway_node *node_at_coords(
 					con = popup->child.view->container;
 					break;
 				}
+#if HAVE_XWAYLAND
+				else if (desc->type == SWAY_SCENE_DESC_XWAYLAND_UNMANAGED) {
+					struct sway_xwayland_unmanaged *unmanaged = desc->data;
+
+					if (unmanaged->xwayland_view) {
+						con = unmanaged->xwayland_view->view.container;
+						break;
+					}else{
+						// we are un an unmanaged surface, but we don't know
```

Comment:
Don't know how to parse this comment, typo?

Reply:
Lol, looks like I got distracted here. The comment is unfinished and a typo definitely does not help